server/cp/planning_exports/common.py

```diff
@@ -225,12 +225,20 @@ def get_event_formatted_dates(event: Dict[str, Any]) -> str:
     tz = pytz.timezone(tz_name)
 
     if all_day:
-        # all day
-        return date_short(start)
+        # All day event
+        return (
+            date_short(start)
+            if start.date() == end.date()
+            else "{} - {}".format(date_short(start), date_short(end))
+        )
 
     if no_end_time:
-        # no end time
-        return "{} {}".format(time_short(start, tz), date_short(start, tz))
+        # no end time event
+        return (
+            "{} {}".format(time_short(start, tz), date_short(start, tz))
+            if start.date() == end.date()
+            else "{} - {}".format(date_short(start, tz), date_short(end, tz))
```

Comment:
in this case there should be start date and time - end date

Reply:
and in general it would be good to use specific format for date time because in some languages time comes after date, so ideally this would be:
```py
    return (
            format_datetime(start, tz)
            if start.date() == end.date()
            else "{} - {}".format(format_datetime(start, tz), format_date(end))
    )
```
also notice not using `tz` for the end, `no_end_time` means we only use the date so there should be no conversion